docs/install_guides/installing-openvino-docker-linux.md

```diff
@@ -1,68 +1,64 @@
-# Install Intel® Distribution of OpenVINO™ toolkit for Linux* from a Docker* Image {#openvino_docs_install_guides_installing_openvino_docker_linux}
+# Install Intel® Distribution of OpenVINO™ toolkit for Linux from a Docker Image {#openvino_docs_install_guides_installing_openvino_docker_linux}
 
-The Intel® Distribution of OpenVINO™ toolkit quickly deploys applications and solutions that emulate human vision. Based on Convolutional Neural Networks (CNN), the toolkit extends computer vision (CV) workloads across Intel® hardware, maximizing performance. The Intel® Distribution of OpenVINO™ toolkit includes the Intel® Deep Learning Deployment Toolkit.  
-
-This guide provides device specifics for a Docker* image creation with Intel® Distribution of OpenVINO™ toolkit for Linux* and its further usage.  
+This guide provides steps on creating a Docker image with Intel® Distribution of OpenVINO™ toolkit for Linux* and using the image on different devices. 
 
 ## System Requirements
 
-**Target Operating Systems**
-
-- Ubuntu\* 18.04 long-term support (LTS), 64-bit
-- Ubuntu\* 20.04 long-term support (LTS), 64-bit
-- CentOS\* 7
-- Red Hat\* Enterprise Linux* 8 (64 bit)
-
-**Host Operating Systems**
+@sphinxdirective
+.. tab:: Target Operating Systems
 
-- Linux
+  * Ubuntu 18.04 long-term support (LTS), 64-bit
+  * Ubuntu 20.04 long-term support (LTS), 64-bit
+  * Red Hat Enterprise Linux 8, 64-bit
 
-## Prebuilt images
+.. tab:: Host Operating Systems
 
-Prebuilt images are available on:
-
-- [Docker Hub](https://hub.docker.com/u/openvino)
-- [Red Hat* Quay.io](https://quay.io/organization/openvino)
-- [Red Hat* Ecosystem Catalog](https://catalog.redhat.com/software/containers/intel/openvino-runtime/606ff4d7ecb5241699188fb3)
+  * Linux
+  * Windows Subsystem for Linux 2 on CPU or GPU
+  * macOS on CPU only
+  
+  To launch a Linux image on Windows Subsystem for Linux 2, note the following things:
 
-## Build a Docker* Image
+  - Only Windows 10 with 21H2 update or above installed and Windows 11 are supported.
+  - Intel GPU driver on Windows host with version 30.0.100.9684 or above need be installed. Please see [this article](https://www.intel.com/content/www/us/en/artificial-intelligence/harness-the-power-of-intel-igpu-on-your-machine.html#articleparagraph_983312434) for more details.
+  - From 2022.1 release, the Docker images contain preinstalled recommended version of OpenCL Runtime with WSL2 support.
```

Comment:
This prerequisites is required only to launch inference on a GPU device. Maybe add something like "with enabled GPU support"?

Reply:
Then I think it's better to still keep this under "Running the Image on GPU" part. Updated. :)